docs/install/amazon-web-services-eks.md

```diff
@@ -0,0 +1,106 @@
+
+* Your EKS clusters use [Ubuntu for EKS](https://cloud-images.ubuntu.com/docs/aws/eks/) as the default node operating system with an optimised kernel. For the kernel module `tcm_loop`, the package `linux-modules-extra-$(uname -r)` is additionally required on each of the nodes - this can be installed automatically by adding it to the node's user data as in the example below.
+
+In this example, we have used [eksctl](https://eksctl.io/introduction/) to create a cluster with 3 nodes of size `t3.large` running Ubuntu for EKS in the `eu-west-2` region. We have provided `100 GB` of disk space for each node - note that by default, Ondat will store data locally in the node's file system under the path `/var/lib/storageos` on each node in [hyperconverged mode](/docs/concepts/nodes/#hyperconverged-mode). In a production infrastructure, we would likely create multiple EBS Volumes tweaked for performance or use ephemeral SSD storage and [mount our volumes under data device directories](/docs/concepts/volumes/) with some additions to user data. We would also implement some form of snapshots or backup of these underlying volumes to ensure continuity in a disaster scenario.
+
+```yaml
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: ondat-cluster
+  region: eu-west-2
+
+managedNodeGroups:
+  - name: ondat-ng
+    minSize: 3
+    maxSize: 3
+    instanceType: t3.large
+    ami: ami-0cb2cb474d9e4e075
+    labels: {ondat: node}
+    volumeSize: 100
+    volumeName: /dev/xvda
+    volumeEncrypted: true
+    disableIMDSv1: true
+    overrideBootstrapCommand: |
+      #!/bin/bash
+      mkdir -p /var/lib/storageos
+      echo "/dev/nvme1n1 /var/lib/storageos ext4 defaults,discard 0 1" >> /etc/fstab
+      mkfs.ext4 /dev/nvme1n1
+      mount /var/lib/storageos
+      sudo apt-get update
+      sudo apt-get install -y linux-modules-extra-$(uname -r)
+      /etc/eks/bootstrap.sh ondat-cluster
+```
+
+> ⚠️ With the above configuration, volumes will be deleted when the nodes they
+> are attached to are terminated. Be sure to keep snapshots, for example using
+> [Data Lifecycle Manager](https://aws.amazon.com/blogs/storage/automating-amazon-ebs-snapshot-and-ami-management-using-amazon-dlm/)
+
+## Procedure
+
+### Step 1 - Conducting Preflight Checks
+
+Run the following command to conduct preflight checks against the EKS cluster to validate that Ondat prerequisites have been met before attempting an installation.
+
+```bash
+kubectl storageos preflight
+```
+
+### Step 2 - Installing Ondat
+
+1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance.
+
+```bash
+export STORAGEOS_USERNAME="admin"
+export STORAGEOS_PASSWORD="password"
+```
+
+2. Run the following  `kubectl-storageos` plugin command to install Ondat.
+
+```bash
+kubectl storageos install \
+  --include-etcd \
```

Comment:
I prefer to put in explicitly the etcd storage class to avoid surprises. EKS clusters use gp2 as default. It might not be what the customer would choose if they knew that etcd uses a SC.

Reply:
Done! Great shout.